utils/build.ps1

```diff
@@ -662,15 +662,15 @@ function Get-TargetInfo([Hashtable] $Platform) {
   # Cache the result of "swiftc -print-target-info" as $Platform.Cache.TargetInfo
   $CacheKey = "TargetInfo"
   if (-not $Platform.Cache.ContainsKey($CacheKey)) {
-    Invoke-IsolatingEnvVars {
+    [void](Invoke-IsolatingEnvVars {
```

Comment:
Why the explicit cast to `void`?

Reply:
Powershell is weird :-)

To quote from the documentation:
> In PowerShell, the results of each statement are returned as output, even without a statement that contains the return keyword. Languages like C or C# return only the value or values that are specified by the return keyword.

The upshot is that `Get-TargetInfo` was returning the output from `Invoke-IsolatingEnvVars`, which isn't what's wanted here. Casting to `void` fixes that.